webpack/components/Table/TableWrapper.js

```diff
@@ -169,7 +171,7 @@ const TableWrapper = ({
                       pageRowCount,
                     }
                 }
-              totalCount={total}
+              totalCount={Number(metadata?.selectable ?? 0)}
```

Comment:
Why `0` and not `total` ? If an API doesn't implement `selectable` for some reason, I wouldn't want it to default to 0..